client/reader/following/view-preference.ts

```diff
@@ -0,0 +1,24 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useDispatch, useSelector } from 'calypso/state';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { getPreference } from 'calypso/state/preferences/selectors';
+
+export const READER_FOLLOWING_VIEW_PREFERENCE = 'reader_following_view';
+export type ReaderFollowingView = 'recent' | 'stream';
+export const DEFAULT_VIEW: ReaderFollowingView = 'stream';
+
+export const useFollowingView = () => {
+	const dispatch = useDispatch();
+	const currentView = useSelector(
```

Comment:
For improving types.

```suggestion
	const currentView: ReaderFollowingView = useSelector(
```

Reply:
Thanks! Implemented in: https://github.com/Automattic/wp-calypso/pull/96798/commits/7afcdd3049f33eca628601266c5f5ba73943c8ac